addons/mod_tool/dock.gd

```diff
@@ -13,6 +13,7 @@ onready var popup := $"%Popup"
 onready var create_mod := $"%CreateMod"
 onready var label_output := $"%Output"
 onready var mod_id := $"%ModId"
+onready var v_box = $"TabContainer/Mod Manifest/ScrollContainer/VBox"
```

Comment:
`var manifest_input_vbox` ? otherwise it's just a random vbox without context

Reply:
Done 👍 Will be moved to a separate file soon™.